src/06-loops.js

```diff
@@ -25,7 +33,15 @@ function rangeSum(min, max) {}
  *  rangeOdd(10, 15);
  *  //> [ 15, 13, 11 ]
  */
-function rangeOdd(min, max) {}
+function rangeOdd(min, max) {
+  let arr = []
+  for (let i = max; i >= min; i --){
```

Comment:
Let's start trying to get into the habit of ending our statements with `;` s.  We don't need to do this for code blocks - `{}`, but for statements it helps the reader and down the road can help with code linters.